### ISSUE TITLE
[$10][MSFT-146] [Challenge Feed] User can't focus on the user avatar using tab key

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2838]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges
2. Press Tab key until the user avatar is selected

**Expected Result**
User should be able to focus on the user avatar menu

**Actual Result**
User can't focus on the user avatar using tab key

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: AA
**WCAG Success Criterion**: 2.4.7 Focus Visible
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F53

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096054 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096054 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096054 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu this PR also covers this https://github.com/topcoder-platform/community-app/pull/2933

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2838 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2838 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2838 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2838 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097355 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097355 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@codeMinter PR updated https://github.com/topcoder-platform/community-app/pull/2933

**Comment by drasticdpk** [comment]
Verified on
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097355<br/><br/>```This is an automated message for crazyk via Topcoder X```
